openreview/stages/arr_content.py

```diff
@@ -360,50 +373,38 @@
             }
         },
         "description": "Upon submitting this paper, authors agree to allow us to share their submission details (such as title, author names, and potentially abstract) with program committees from other conference venues for the purpose of verifying compliance with submission requirements.",
-        "order": 25
+        "order": 26
     },
-    "A1": {
+    "A1_limitations_section": {
         "value": {
             "param": {
-                "input": "radio",
+                "input": "checkbox",
                 "enum": [
-                    "yes",
-                    "no",
-                    "n/a"
+                    "This paper has a limitations section."
```

Comment:
if this is the only option and it is not optional, does it mean all the submissions should say "this paper has a limitations section"?

Reply:
ok, They must select it otherwise it will be desk rejected.

Reply:
Yeah I guess they just want the authors to have to explicitly check that off